www/class/centreonSession.class.php

```diff
@@ -56,6 +56,10 @@ public function start($flag = 0)
 
     public function stop()
     {
+        // delete the current cookie using the 'PHP session expiration time' value
+        setcookie(session_name(), "", time() - 31536000);
```

Comment:
a bit weird to delete cookie like that.
cookie will be deleted by garbage collector.